eu/ja-jp/content/topics/end-user/accessibility.htm

```diff
@@ -142,7 +142,7 @@
                                     <div class="nav-search row">
                                         <form class="search" action="#">
                                             <div class="search-bar search-bar-container needs-pie">
-                                                <input class="search-field needs-pie" type="search" aria-label="検索フィールド" placeholder="検索" />
+                                                <input class="search-field needs-pie" type="search" aria-label="［Search（検索）］フィールド" placeholder="検索" />
```

Comment:
@pengyao-okta The aria label was changed here, unnecessarily. This is not a UI control element, and thus does not need the bilingual rendering